### PR TITLE
MODINV-147 Inventory App: Assign tags to item record

### PR DESCRIPTION
**Pull request by stanislav-mureiko**
## Purpose
Due to https://issues.folio.org/browse/MODINV-147 we would like to introduce tags for item entities.

## Approach
* updated item with tags
* updated interfaces versions
* updated items implementation to include tags
* updated item tests accordingly

### CODE REVIEW

**Comment by stanislav-mureiko** [comment]
should be merged after https://github.com/folio-org/mod-inventory-storage/pull/304

**Comment by kostyantyn-kh** [comment]
Please resolve conflicts

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "9.1",
+      "version": "9.3",
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: Why has this been incremented by two minor versions?

File: src/test/java/api/items/ItemApiExamples.java [hunk 2]
```diff
@@ -381,6 +404,12 @@ public void canUpdateExistingItem()
     assertThat(getResponse.getStatusCode(), is(200));
     JsonObject updatedItem = getResponse.getJson();
 
+    assertThat(updatedItem.containsKey(Item.TAGS_KEY), is(true));
+    final JsonObject tags = updatedItem.getJsonObject(Item.TAGS_KEY);
```
Comment (marcjohnson-kint) [inline src/test/java/api/items/ItemApiExamples.java]: Personally, I might include whitespace between these three blocks of code, to better separate them

**Comment by marcjohnson-kint** [comment]
> ## Purpose
> Due to https://issues.folio.org/browse/MODINV-147 we would like to introduce tags for instance entities.

Should this say items instead of instances?

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -136,7 +136,7 @@
     },
     {
       "id": "inventory-batch",
-      "version": "0.2",
+      "version": "0.4",
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: Are items included in the inventory-batch interface?

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (stanislav-mureiko) [reply descriptors/ModuleDescriptor-template.json]: PRs related assigning tags to instance records changed version of inventory interface from 9.1 to 9.2. And PRs about assigning tags to item records will update version to 9.3.

File: descriptors/ModuleDescriptor-template.json (hunk 3 above)
Comment (stanislav-mureiko) [reply descriptors/ModuleDescriptor-template.json]: > Are items included in the inventory-batch interface?

Sorry it's mistake. I'll fix it.

**Comment by stanislav-mureiko** [comment]
> > ## Purpose
> > Due to https://issues.folio.org/browse/MODINV-147 we would like to introduce tags for instance entities.
> 
> Should this say items instead of instances?

File: src/test/java/api/items/ItemApiExamples.java [hunk 4]
```diff
@@ -140,6 +153,14 @@ public void canCreateItemWithAnID()
 
     JsonObject materialType = createdItem.getJsonObject("materialType");
 
+    assertThat(createdItem.containsKey(Item.TAGS_KEY), is(true));
```
Comment (marcjohnson-kint) [inline src/test/java/api/items/ItemApiExamples.java]: Personally, I might split up these assertions into logical units using some whitespace.

**Pull request merged**